src/fish.cpp

```diff
@@ -57,6 +57,14 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA
 #define PATH_MAX 4096
 #endif
 
+// container to hold the options specified within the command line
+class command_line_switches_t {
+public:
+    std::vector<std::string> cmds;              // commands to be executed in place of interactive shell
```

Comment:
Maybe `batch_cmds` and `postconfig_cmds`? Not essential but the use of a descriptive adverb on one member but not the other caused me to double-check my comprehension (mostly elsewhere in the code).